packages/cadl-ranch-specs/http/README.md

```diff
@@ -25,3 +25,5 @@ The tests in the http/models directory describe scenarios for generating models
 - [nested-models](https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/models/nested-models) - Generate, send, and receive input, output, and round-trip models with required nested model properties.
 - [readonly-properties](https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/models/readonly-properties) - Generate, send, and receive input, output, and round-trip models with readonly properties.
 - [inheritance](https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/models/inheritance) - Generate, send, and receive round-trip models polymorphic models.
+- [inheritance-complex](https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/models/inheritance-complex) - Generate, send, and receive complex polymorphic models.
```

Comment:
I moved them to the `inheritance`.
Since the tests in original `inheritance-complex` can fully cover the existing `inheritance`, I think it's fine to remove the existing model&operations in the heritance?